blocks/milestone/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+const { __ } = wp.i18n;
+const { registerBlockType } = wp.blocks;
+
+/**
+ * Internal dependencies
+ */
+import blockAttributes from './data/attributes';
+import GiveLogo from '../components/logo';
+import Milestone from './edit';
+
+/**
+ * Required styles (both common and editor styles)
+ */
+import './common.scss';
+import './editor.scss';
+
+/**
+ * Register Block
+ */
+
+export default registerBlockType( 'give/milestone', {
+	title: __( 'Milestone' ),
+	description: __( 'The GiveWP Milestone block inserts an progress bar indicating donations raised across multiple forms. Each Milestone\'s presentation can be customized below.' ),
+	category: 'give',
+	icon: <GiveLogo color="grey" />,
+	keywords: [
+		__( 'donation' ),
+		__( 'milestone' ),
```

Comment:
@henryholtgeerts Text domain is missing

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5224/commits/0311eb6accaff1dfae9d6d85043cbd68e4b01bb9